_pages/en/esdc-it-strategy.md

```diff
@@ -28,11 +28,14 @@ In addition, ESDC has started investing in a [Business Delivery Modernization](h
 
 ## Problem Statement
 
-As explained in [ESDC's Medium-Term Planning](http://dialogue/grp/IITB-DGIIT-Gov-New-Nouveau/Briefings/MTP-Initiative-Mapping_2018.docx), ESDC operates a portfolio of approximately 500 applications, 101 of which are mission critical.
+Moving to digital is creating a world that is highly complex.
```

Comment:
I see where you are getting at and I agree... but we need to work on simplifying the complexity as much as possible. Loosely coupled parts, clear documented APIs, automated tests... We could reword "highly" to something less strong, at your discretion.